### ISSUE TITLE
Suggestion: split the mod into several parts

### ISSUE THREAD

**Issue by Youda008** [saegiru/UltimateDoomVisor#3]
Hello. I have kinda unorthodox request here.

This mod has grown quite a bit, far beyond being just a helmet, it has a lot of gameplay features, even own difficulty levels. The idea is, some of those features would be useful alone, without the helmet, and they have no alternative between the doom mods.

So here is my proposal. Could you please split the UDV into 2 independent files containing the following?

**Helmet itself**

- all visual and audio features and settings
- radar, flashlight, batteries

**Gameplay features**

- starting armor
- bleeding, energized, aim momentum
- _emergency medial system_ (on the edge, to be discussed)
- custom difficulties
- autosaves

The motivation is that i would like to use some of the gameplay features (especially autosaves and starting armor) even in Vanialla doom and there is no other mod that would do just that. The higher granularity (more number of smaller mods) the more we can customize our Doom how we want.
Please consider it.

**Comment by Scallysnix** [comment]
I agree. I often install this mod, not for the helmet, for the gameplay enhancements specifically. I would love to see these features expanded on.
